Blish HUD/GameServices/Graphics/ManualUISize.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.ComponentModel;
+
+namespace Blish_HUD.Graphics {
+    public enum ManualUISize : int {
+        [Description("Automatic - Sync With Game")]
```

Comment:
For the sake of consistency, let's just have this read `Sync With Game`.

Reply:
I have copied this from `DPIMethod` should we change it there too then?

Reply:
Upon further inspection I'd suggest changing the name of the enum to `UISizeMethod` too?

Reply:
I probably am just misremembering things.  I'll check it tonight.  It's probably all fine to leave as is if you copied it over.  :D